pyccel/codegen/utilities.py

```diff
@@ -111,7 +112,10 @@ def copy_internal_library(lib_folder, pyccel_dirpath, extra_files = None):
     """
     # get lib path (stdlib_path/lib_name or ext_path/lib_name)
     if lib_folder in external_libs:
-        lib_path = os.path.join(ext_path, external_libs[lib_folder], lib_folder)
+        if external_libs[lib_folder] == "STC_Extensions":
```

Comment:
STC_Extensions is an internal library not an external library. Is it not possible to handle it in the same way as other external libraries?

Reply:
The reason for this is to include the two folders, `STC` and `STC_extensions`, in the same way.

Reply:
Yes that is my point. As they are conceptually different I don't see why they should be included in the same way. I think STC_extensions should be handled like every other folder in pyccel/stdlib

Reply:
Fixed  : 96aec13